### PR TITLE
LPD-7009 read and validate script element attributes from .yaml file for deployed client extension

### PR DESCRIPTION
**Pull request by thiago-buarqque**
This is the second part of #492 ([part 1](https://github.com/liferay-platform-experience/liferay-portal/pull/507)). 

Adding validation and mapping to `scriptElementAttributes`

### CODE REVIEW

**Comment by thiago-buarqque** [comment]
ci:test:sf

File: modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json [hunk 1]
```diff
@@ -217,6 +217,33 @@
...
+										"allOf": [
+											{
+												"required": [
+													"src"
+												]
+											}
+										]
+									},
+									"title": "Script Element Attributes",
+									"type": "object"
```
Comment (thiago-buarqque) [inline modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json:243]: There is a whole discussion about this part of the schema. Refer to https://github.com/liferay-platform-experience/liferay-portal/pull/492/commits/d52d7e41743ccb4359c07f589f5e44b87a2eeff2#r1509646275

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Review by ethib137 — APPROVED** [review]
LGTM

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Comment by ethib137** [comment]
@thiago-buarqque please send to https://github.com/liferay-devtools/liferay-portal/pulls Thank you.

**Comment by thiago-buarqque** [comment]
Resent to liferay-devtools [#35](https://github.com/liferay-devtools/liferay-portal/pull/35)

**Pull request closed without merging**